guides/v2.1/install-gde/prereq/security.md

```diff
@@ -13,12 +13,11 @@ functional_areas:
   - Setup
 ---
 
-<h2 id="install-prereq-selinux">SELinux</h2>
-<a href="http://selinuxproject.org/page/Main_Page" target="_blank">Security Enhanced Linux (SELinux)</a> enables CentOS and Ubuntu administrators greater access control over their servers. If you're using SELinux *and* Apache must initiate a connection to another host, you must run the commands discussed in this section.
+## SELinux {#install-prereq-selinux}
```

Comment:
Make these changes on version 2.3 as well, as it is not symlinked to prior versions.